checkov/terraform/checks/graph_checks/aws/S3BucketEncryption.yaml

```diff
@@ -12,23 +12,41 @@ definition:
       value:
         - "aws:kms"
         - "AES256"
-    - and:
-      - cond_type: filter
-        attribute: resource_type
-        operator: within
-        value:
-          - aws_s3_bucket
-      - cond_type: connection
-        resource_types:
-          - aws_s3_bucket
-        connected_resource_types:
-          - aws_s3_bucket_server_side_encryption_configuration
-        operator: exists
-      - cond_type: attribute
-        resource_types:
-          - aws_s3_bucket_server_side_encryption_configuration
-        attribute: rule.apply_server_side_encryption_by_default.sse_algorithm
-        operator: within
-        value:
-          - "aws:kms"
-          - "AES256"
+    - or:
```

Comment:
you can remove this `or` and move the `and`s one level left

Reply:
done :)